vips/vip-131/vip-131.ts

```diff
@@ -5,23 +5,31 @@ import { makeProposal } from "../../src/utils";
 
 export const FEE_OUT = 100; // 10bps
 export const FEE_IN = 0;
+export const BASE_RATE_MANTISSA = parseUnits("4", 18); // 4%
```

Comment:
```suggestion
export const BASE_RATE_MANTISSA = parseUnits("0.04", 18); // 4%
```

Reply:
I would add a check in the simulation, before executing the VIP, to verify the current value. This way you will be aware of the units (in this case, the current value is 0.01e18)

Reply:
Correct